src/seraphis_main/scan_machine.cpp

```diff
@@ -87,18 +96,20 @@ static std::uint64_t get_reorg_avoidance_depth(const std::uint64_t reorg_avoidan
         "reorg avoidance increment.");
 
     // 3. 10 ^ (num requests - 1) * increment
-    return math::saturating_mul(math::uint_pow(10, num_reorg_avoidance_backoffs - 1), reorg_avoidance_increment, -1);
+    const std::uint64_t exponent = math::saturating_sub(num_reorg_avoidance_backoffs, 1, 0);
+    return math::saturating_mul(math::uint_pow(10, exponent), reorg_avoidance_increment, -1);
 }
 //-------------------------------------------------------------------------------------------------------------------
 //-------------------------------------------------------------------------------------------------------------------
 static std::uint64_t get_estimated_start_scan_index(const std::uint64_t reorg_avoidance_increment,
+    const bool force_reorg_avoidance_increment,
     const std::uint64_t num_reorg_avoidance_backoffs,
     const std::uint64_t lowest_scannable_index,
     const std::uint64_t desired_start_index)
 {
     // 1. set reorg avoidance depth
     const std::uint64_t reorg_avoidance_depth{
-            get_reorg_avoidance_depth(reorg_avoidance_increment, num_reorg_avoidance_backoffs)
+            get_reorg_avoidance_depth(reorg_avoidance_increment, force_reorg_avoidance_increment, num_reorg_avoidance_backoffs)
```

Comment:
```suggestion
            get_reorg_avoidance_depth(reorg_avoidance_increment,
                force_reorg_avoidance_increment,
                num_reorg_avoidance_backoffs)
```
Your font size is too small :p